packages/invoices/src/OutboundShipment/DetailView/SidePanel.tsx

```diff
@@ -106,6 +111,69 @@ const RelatedDocumentsSection: FC = () => {
   );
 };
 
+export const PricingSection = () => {
+  const t = useTranslation('distribution');
+  const serviceLineModal = useToggle(false);
+  const { c } = useCurrency();
+  const { pricing } = useOutboundFields(['pricing']);
+
+  return (
```

Comment:
This looks OK. Not great - maybe OK for now?

Reply:
My helpful comment is that I think the pencil is too big 😄 

I've played around a bit and put a suggestion in 529449ae
I know that using margin is well dodgy.. looks nicer with a bit of spacing is all.

Reply:
Thanks 😁